app/src/main/java/com/example/tasky/core/di/RetrofitModule.kt

```diff
@@ -34,6 +35,7 @@ class RetrofitModule {
                     .build()
                 it.proceed(request)
             }
+            .addInterceptor(jwtInterceptor)
```

Comment:
This wont be used anywhere else
So you could just avoid having it in parameters, and just create the instance here
Since its a @Singleton

Reply:
Its either having the interceptor as a paremeter (since hilt provides there an instance) or having the UserCache as parameter to create an instance there

Reply:
Yea you are right!

Totally missed that